src/cmd/manage_nsfs.js

```diff
@@ -414,8 +416,14 @@ async function fetch_account_data(argv, from_file) {
     // secret_key as SensitiveString
     data.access_keys[0].secret_key = _.isUndefined(data.access_keys[0].secret_key) ? undefined :
         new SensitiveString(String(data.access_keys[0].secret_key));
-    // allow_bucket_creation infer from new_buckets_path
-    data.allow_bucket_creation = !_.isUndefined(data.nsfs_account_config.new_buckets_path);
+    // allow_bucket_creation either set by user or infer from new_buckets_path
+    if (_.isUndefined(argv.allow_bucket_creation)) {
+        data.allow_bucket_creation = !_.isUndefined(data.nsfs_account_config.new_buckets_path);
+    } else if (typeof argv.allow_bucket_creation === 'string') {
+        data.allow_bucket_creation = argv.allow_bucket_creation.toLowerCase() === 'true';
+    } else {
+        data.allow_bucket_creation = Boolean(argv.allow_bucket_creation);
```

Comment:
@shirady @romayalon Are we considering only `allow_bucket_creation` flag to decide for enabling bucket creation, not with `new_buckets_path`, So what will happen when user set `allow_bucket_creation` true and `new_buckets_path` empty and try to create a bucket using S3, What will be the bucket's storage path.

Reply:
Looking at the current code it should be fine:
https://github.com/noobaa/noobaa-core/blob/7d2248c73f665060fc43b6c84de3ca38506c76ef/src/sdk/bucketspace_fs.js#L247-L252
In this case the `allow_bucket_creation` is true so it will not throw the first error, but `nsfs_account_config.new_buckets_path` is not defined so it should throw the second error 'MISSING_NSFS_ACCOUNT_CONFIGURATION'.